{"name": "test_login[login_case_fixture1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 20000 == 10001\n  +20000\n  -10001", "trace": "self = <TestCases.test_login.TestLogin object at 0x000002BD24B1D310>\nlogin_case_fixture = {'api_type': 'login', 'case_id': 'login_002', 'data': {'password': '12345678'}, 'expect': 10001, ...}\n\n>   ???\nE   assert 20000 == 10001\nE     +20000\nE     -10001\n\nD:\\workspace\\PycharmProjects\\job\\TestCases\\test_login.py:17: AssertionError"}, "parameters": [{"name": "login_case_fixture", "value": "{'id': 2, 'case_id': 'login_002', 'title': '登录失败_用户名为空', 'api_type': 'login', 'uri': '/loginWithJwt', 'method': 'get', 'data': {'password': '12345678'}, 'expect': 10001}"}], "start": 1710855514070, "stop": 1710855514233, "uuid": "e82cbbf2-9085-41a5-a937-fcd51a5c220a", "historyId": "7de0438e751bcfcc65745259c4ec4eb3", "testCaseId": "834c0cf788ac59bcddad40943dc8846f", "fullName": "TestCases.test_login.TestLogin#test_login", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-K98MN2L"}, {"name": "thread", "value": "8236-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}